src/components/BlockingViews/FullPageNotFoundView.js

```diff
@@ -48,9 +51,10 @@ const defaultProps = {
     subtitleKey: 'notFound.pageNotFound',
     linkKey: 'notFound.goBackHome',
     shouldShowBackButton: true,
-    shouldShowBackHomeLink: false,
+    shouldShowLink: false,
     shouldShowCloseButton: true,
     onBackButtonPress: () => Navigation.dismissModal(),
+    onLinkPress: () => Navigation.dismissModal(),
```

Comment:
Should we pass `true` to open the LHN drawer? It's what we had before so it's probably right, but I'm actually not sure, what do you guys think?

Reply:
Oh, yes. need to pass `true` as it was like that earlier.